src/ngraph/frontend/onnx_import/op/quant_conv.cpp

```diff
@@ -0,0 +1,248 @@
+
+                                convolution_nodes.push_back(
+                                    ngraph::builder::quantization::QuantizedLinearConvolution(
+                                        sliced_data,
+                                        sliced_filters,
+                                        strides,
+                                        filter_dilations,
+                                        padding_below,
+                                        padding_above,
+                                        data_dilations,
+                                        op_scale.data_scale,
+                                        op_scale.filter_scale,
+                                        op_scale.output_scale));
+                            }
+                            std::size_t concatenation_axis = 1;
+                            return std::make_shared<ngraph::op::Concat>(convolution_nodes,
+                                                                        concatenation_axis);
+                        }
+                        else
+                        {
+                            return ngraph::builder::quantization::QuantizedLinearConvolution(
+                                data,
+                                filters,
+                                strides,
+                                filter_dilations,
+                                padding_below,
+                                padding_above,
+                                data_dilations,
+                                op_scale.data_scale,
+                                op_scale.filter_scale,
+                                op_scale.output_scale);
+                        }
+                    }
+
+                    std::shared_ptr<ngraph::Node>
+                        make_ng_quant_conv_bias(const std::shared_ptr<ngraph::Node>& data,
+                                                const std::shared_ptr<ngraph::Node>& filters,
+                                                const std::shared_ptr<ngraph::Node>& bias,
+                                                const ngraph::Strides& strides,
+                                                const ngraph::Strides& filter_dilations,
+                                                const ngraph::CoordinateDiff& padding_below,
+                                                const ngraph::CoordinateDiff& padding_above,
+                                                const ngraph::Strides& data_dilations,
+                                                int groups,
+                                                const OpScale& op_scale,
+                                                const OpZeroPoint& op_zero_point)
+                    {
+                        if (groups > 1)
+                        {
+                            throw ngraph::onnx_import::error::NotSupported(
+                                "No support for quantized group conv+bias.");
```

Comment:
Please check whether this is still unsupported?

Reply:
@adstraw should we try to fix that or it will be supported from different place?